cardano-crypto-class/src/Cardano/Crypto/DSIGN/Mock.hs

```diff
@@ -158,13 +155,3 @@ data VerificationFailure
       , vErrCallStack :: String
       }
   deriving Show
-
-mockSign :: ToCBOR a => a -> SignKeyDSIGN MockDSIGN -> SigDSIGN MockDSIGN
-mockSign a (SignKeyMockDSIGN n) = SigMockDSIGN (castHash (hash a)) n
```

Comment:
This is used in some of the consensus testing code.

Reply:
restored